notebooks/Tutorial8-Time_series.py

```diff
@@ -0,0 +1,287 @@
+#
+# Note that it is possible to generate avatars of data containing only time series dataset (i.e. without any vanilla data). In this case, dataset_id can be set to `None`
+
+job = client.jobs.create_avatarization_with_time_series_job(
+    AvatarizationWithTimeSeriesJobCreate(
+        parameters=AvatarizationWithTimeSeriesParameters(
+            vanilla_dataset_id=dataset_vanilla.id,
+            k=5,
+            time_series=[
+                AvatarizationTimeSeriesParameters(
+                    dataset_id=dataset_ts.id,
+                    projection_parameters=FPCAParameters(
+                        nf=10,
+                    ),
+                )
+                for dataset_ts in datasets_ts
+            ],
+        )
+    ),
+    timeout=50,
+)
+job = client.jobs.get_avatarization_time_series_job(job.id, timeout=60)
+
+# ### Retrieve avatars
+#
+# Once generated, the avatar dataset are stored on the server and can be retrieved using theuir dataset ID. The dataset ID  for each avatar dataset is contained in `job.result.datasets`.
+
+# +
+print(
+    f"There are {len(job.result.datasets)} avatars datasets following anonymization.\n"
+)
+
+print(f"Their UUIDs are : {[x.avatars_dataset.id for x in job.result.datasets]}\n")
+
+originals_to_avatars = {
+    x.original_id: x.avatars_dataset.id for x in job.result.datasets
+}
+print(f"The mapping original to avatar is: {originals_to_avatars}")
+# -
+
+# ### Checking your avatars: vanilla data
+
+avatars_vanilla_df = client.pandas_integration.download_dataframe(
+    job.result.datasets[0].avatars_dataset.id
+)
+
+avatars_vanilla_df.head()
+
+# ### Checking your avatars: `sensor1` and `sensor2`
+
+avatars_ts1_df = client.pandas_integration.download_dataframe(
+    job.result.datasets[1].avatars_dataset.id
+)
+
+plot = plot_series(
+    df := avatars_ts1_df,
+    variable_to_plot="sensor1",
+    id_variable="id",
+    time_variable="t",
+    proportion_to_plot=1.0,
+    n_series_to_plot=None,
+    figsize=(14, 8),
+)
+plot.show()
+
+plot = plot_series(
+    df := avatars_ts1_df,
+    variable_to_plot="sensor2",
+    id_variable="id",
+    time_variable="t",
+    proportion_to_plot=1.0,
+    n_series_to_plot=None,
+    figsize=(14, 8),
+)
+plot.show()
+
+# ### Checking your avatars: `sensor3`
+
+avatars_ts2_df = client.pandas_integration.download_dataframe(
+    job.result.datasets[2].avatars_dataset.id
+)
+
+plot = plot_series(
+    df := avatars_ts2_df,
+    variable_to_plot="sensor3",
+    id_variable="id",
+    time_variable="t",
+    proportion_to_plot=1.0,
+    n_series_to_plot=None,
+    figsize=(14, 8),
+)
+plot.show()
+
+# As for any data, privacy metrics need to be computed on the output time series data to confirm that it is anonymous.
```

Comment:
I think we should add a section to show how to compute metrics and get the results.

Reply:
it was meant to be there, got lost somehow